src/tree/test_resolution.rs

```diff
@@ -0,0 +1,66 @@
+use super::*;
+
+#[test]
+
+fn test_exclusion_list() {
+    for ciphersuite in Config::supported_ciphersuites() {
+        const NODES: usize = 31;
+        const FULL_RESOLUTION: &[usize] =
+            &[0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30];
```

Comment:
Can you add some description that tells me what exactly the test is (supposed to be) doing?